types/braintree-web/index.d.ts

```diff
@@ -489,7 +489,13 @@ declare namespace braintree {
     placeholder?: string;
     type?: string;
     formatInput?: boolean;
+    maskInput?: any;
```

Comment:
Let's make this field more specific?

```ts
interface HostedFieldsFieldMaskInput {
    /**
     * The character to use when masking the input.
     * @default '•'
     */
    character?: string;
    /**
     * Only applicable for the credit card field. Whether or not to show the last 4 digits of the card when masking.
     */
    showLastFour?: boolean;
}
```

```ts
interface HostedFieldsField {
    // ...
    /**
     * Enable or disable input masking when input is not focused. If set to `true` instead of an object, the defaults for the `maskInput` parameters will be used.
     */
    maskInput?: boolean | HostedFieldsFieldMaskInput;
    // ...
}
```

Reply:
sure